build-config/scripts/onie-mk-demo.sh

```diff
@@ -25,6 +25,10 @@ if [ "$arch" = "powerpc-softfloat" -o "$arch" = "armv7a" ] ; then
     # Both of these architectures share common installer code as they
     # are both based on u-boot.
     arch_dir="u-boot-arch"
+elif [ "$arch" = "x86_64" -o "$arch" = "armv8a" ] ; then
+    # These architectures share common installer code as they
+    # are based on grub.
+    arch_dir="grub-arch"
```

Comment:
@david56 - this PR is for adding aarch64 (armv8a).  we do not have any armv8a machines in the project yet.  are you saying you expect some future aarch64 machines to use u-boot?

i would really like to avoid that situation -- some aarch64 using u-boot and some using grub.  i would expect all aarch64 machines to be UEFI and grub.